proto/redpanda/api/dataplane/v1alpha2/pipeline.proto

```diff
@@ -400,7 +452,15 @@ message StartPipelineResponse {
 message GetPipelineServiceConfigSchemaRequest {
   enum Format {
     FORMAT_UNSPECIFIED = 0;
-    FORMAT_JSONSCHEMA = 1;
+
+    // The default text format that lists all components.
+    FORMAT_TEXT = 1;
```

Comment:
This looks more like another endpoint. It could be `ListPipelineServiceConfigSupportedComponents`